File: raguilar9.java
//Ryan Aguilar
//CSC321
//Lab 9c

import java.util.Random;
import java.util.Scanner;

public class raguilar9
{
	public static void main(String[] args)
	{
		Scanner keyboard = new Scanner(System.in);

		//Generates a random number between 1 and 21
		Random rand = new Random();
		int randNum = rand.nextInt(1, 21);
		
		//DEBUG
		System.out.printf("DEBUG: CORRECT ANSWER: %d\n", randNum);
		
		System.out.printf("Guess my number.\nEnter any number between 1 and 20: ");
		int userNum = keyboard.nextInt();

		while(userNum < 1 || userNum > 20)
		{
			System.out.printf("Oops, I need a number between 1 and 20: ");
			userNum = keyboard.nextInt();
		}

		if(userNum > randNum)
		{
			System.out.printf("Oops, your number was larger than mine, which was %d. \n", randNum);
		}
		else if(userNum < randNum)
		{
			System.out.printf("Oops, your number was smaller than mine, which was %d. \n", randNum);
		}
		else if(userNum == randNum)
		{
			for(int i = 0; i < userNum; i++)
			{
				System.out.printf("You win, you guessed my number: %d!\n", randNum);
			}
		}

	}
}

File: raguilar9.c
//Ryan Aguilar
//CSC321
//Lab 9c

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void)
{
	int guess = 0;
	int randNum = 0;
	int i;

	srand(time(NULL));
	randNum = 1 + rand() % 20;
	
	printf("DEBUG: CORRECT ANSWER IS %d\n", randNum);
	
	while(guess < 1 || guess > 20)
	{
		printf("Guess my number, it's between 1 and 20: ");
		scanf("%d", &guess);

		if(guess < 1 || guess > 20)
		{
			printf("Error, Try again.");
		}
	}

	if(randNum == guess)
	{
		for(i = 0; i < guess; i++)
		{
			printf("You won and guessed my number correctly!\n");
		}
	}
	else if(randNum > guess)
	{
		printf("\nOops, incorrect number! Your guess was less than my number, which was %d \n", randNum);
	}
	else if(randNum < guess)
	{
		printf("\nOops, incorrect number! Your guess was bigger than my number, which was %d \n", randNum);
	}

	return EXIT_SUCCESS;
}
